### ISSUE TITLE
packetfence-tc.service requires mariadb

### ISSUE THREAD

**Issue by danpoltawski** [inverse-inc/packetfence#3932]
When starting up packetfence, the packetfence-tc service often fails to start due to being unable to connect to maraiadb, looking at the logs:

```
packetfence[847]: FATAL -e(847): unable to connect to database: Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2 "No such file or direc
(pf::db::db_connect)
packetfence[847]: FATAL -e(847): unable to connect to database: Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2 "No such file or direc
(Log::Log4perl::Logger::logcroak)
perl[847]: unable to connect to database: Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2 "No such file or directory") at /usr/local/p
systemd[1]: packetfence-tc.service: main process exited, code=exited, status=255/n/a
systemd[1]: Failed to start PacketFence Traffic Shaping configuration.
```

Looking at
https://github.com/inverse-inc/packetfence/blob/3df5ee08302003b398ac1db15e58a6ce4da1ae28/conf/systemd/packetfence-tc.service#L4

Perhaps a 
```After=packetfence-mariadb.service```

Should be added?

**Comment by julsemaan** [comment]
Are you running a cluster or a standalone server?

Technically in standalone since packetfence-mariadb is part of packetfence-base it should be started before packetfence-tc is started.

If you're in cluster, then its because packetfence-haproxy-db isn't started but we can't add it to the "After" list since not everyone runs a cluster.

The solution could be to get a loop that waits for the DB to be started before loading the rules in like in iptables:
https://github.com/inverse-inc/packetfence/blob/3df5ee08302003b398ac1db15e58a6ce4da1ae28/conf/systemd/packetfence-iptables.service#L10

I'll take care of this in devel, not in maintenance though so it can be tested thoroughly during the next release test phase

**Comment by julsemaan** [comment]
fixed in 2779f0d81671a9a04895977c5d1a9e1ae0034e7c

**Comment by danpoltawski** [comment]
> Are you running a cluster or a standalone server?

Sorry, I didn't see your question. We are running standalone.
